Signal/src/ViewControllers/SafetyNumberConfirmationAlert.swift

```diff
@@ -87,33 +95,31 @@ class SafetyNumberConfirmationAlert: NSObject {
         let dismissAction = UIAlertAction(title: NSLocalizedString("TXT_CANCEL_TITLE", comment: "generic cancel text"), style: .cancel)
         actionSheetController.addAction(dismissAction)
 
-        fromViewController.present(actionSheetController, animated: true)
+        UIApplication.shared.frontmostViewController?.present(actionSheetController, animated: true)
```

Comment:
NBD but I've been wondering if we should have an `OWSAlerts.present(alert, animated)` method to DRY up a tiny bit.